src/components/action-popover/action-popover.component.tsx

```diff
@@ -221,7 +224,7 @@ export const ActionPopover = ({
         "data-element": "action-popover-button",
         ariaAttributes: {
           "aria-haspopup": "true",
-          "aria-label": l.actionPopover.ariaLabel(),
+          "aria-label": ariaLabel || l.actionPopover.ariaLabel(),
```

Comment:
**comment/question** (non-blocking): I can see here you're using an OR operator, which will default to the en-gb locale. Which by default is just `"Action"`, I was just wondering if we shouldn't change this within the PR to be a tad more descriptive, especially as we're leaving the translation in for the Button which opens the popover. Maybe something like `"Open Popover"` etc.

Reply:
While I dislike the "actions" default, I don't feel comfortable changing this as it will change the accessible name for any current implementation where they don't currently providfe their own. Come to think of it, it's probably really not a big deal, but I'm also not sure I want a debate now on what to change it to, as none of us really has a clue what it should be changed to. (It really needs a content designer and we don't have one on the team.) Happy to be persuaded though.